modules/auxiliary/scanner/portscan/ack.rb

```diff
@@ -27,6 +27,8 @@ def initialize
       OptString.new('PORTS', [true, "Ports to scan (e.g. 22-25,80,110-900)", "1-10000"]),
       OptInt.new('TIMEOUT', [true, "The reply read timeout in milliseconds", 500]),
       OptInt.new('BATCHSIZE', [true, "The number of hosts to scan per set", 256]),
+      OptInt.new('DELAY', [true, "The delay between connections, per thread, in milliseconds", 0]),
+      OptInt.new('JITTER', [true, "The delay jitter factor (maximum value by which to +/- DELAY) in milliseconds.", 0]),
```

Comment:
Understood, will do - the only reason I didn't was because not all modules will implement these and I didn't want to cause confusion....is it acceptable to have advanced options that will only be honoured sometimes? :)